docs/articles/sdk/installing-the-sdk.rst

```diff
@@ -11,7 +11,7 @@ Installing the SDK
 ------------------
 
 The SDK file is not executable by default. This needs to be changed with `chmod`. If you downloaded
-the SDK from the internet, the ``<SDK location>`` is typically ``~/Downloads``.
+the SDK from the internet, the ``<SDK location>`` is typically ``~/Downloads``. If you built the sdk, it can be found on ``../build/tmp/deploy/sdk`` 
```

Comment:
>>the SDK from the internet, the ``<SDK location>`` is typically ``~/Downloads``
I don't get it :|

sdk < SDK

Reply:
When you get the sdk from pelux.io/downloads/ it will be on ~/Downloads (i know lol)